internal/events/api/event.go

```diff
@@ -65,3 +65,8 @@ type InfraEnvEvent interface {
 	GetInfraEnvId() strfmt.UUID
 	GetClusterId() *strfmt.UUID
 }
+
+type HostUpdateStatusEvent interface {
+	HostEvent
```

Comment:
If we make sure that every event with an `Info` field implements this interface, then we can make this something like `InfoEvent` and only include `BaseEvent`'s interface here.